project-base/storefront/components/Blocks/StoreList/StoreListItem.tsx

```diff
@@ -0,0 +1,90 @@
+import { ArrowIcon } from 'components/Basic/Icon/ArrowIcon';
+import { OpeningHours } from 'components/Blocks/OpeningHours/OpeningHours';
+import { OpeningStatus } from 'components/Blocks/OpeningHours/OpeningStatus';
+import { LinkButton } from 'components/Forms/Button/LinkButton';
+import { TypeOpeningHoursFragment } from 'graphql/requests/stores/fragments/OpeningHoursFragment.generated';
+import useTranslation from 'next-translate/useTranslation';
+import { Fragment, useEffect, useState } from 'react';
+import { StoreOrPacketeryPoint } from 'utils/packetery/types';
+import { twMergeCustom } from 'utils/twMerge';
+
+export const StoreListItem: FC<{ store: StoreOrPacketeryPoint; isSelected: boolean }> = ({ store, isSelected }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+    const { t } = useTranslation();
+
+    useEffect(() => {
+        setIsExpanded(isSelected);
+    }, [isSelected]);
+
+    const getCurrentOpeningHours = (openingHours: TypeOpeningHoursFragment) => {
```

Comment:
Each time the `StoreListItem` component is rendered, this code is executed. I recommend moving it to a separate file or at least extracting it outside of this component.

Reply:
Moved to separated file.